src/domain/core/src/entities/metadata_chain.rs

```diff
@@ -76,7 +76,10 @@ pub trait MetadataChain: Send + Sync {
     ) -> Result<Multihash, AppendError>;
 
     fn as_object_repo(&self) -> &dyn ObjectRepository;
```

Comment:
I expect `as_object_repo` should be removed completely and `MetadataBlockRepository` would have those operations about `get_size` and `get_bytes`.

Reply:
Done, returned back